stand/libsa/in_cksum.c

```diff
@@ -90,5 +88,5 @@ in_cksum(p, len)
 		sum += v;
 	sum = (sum >> 16) + (sum & 0xffff); /* add in accumulated carries */
 	sum += sum >> 16;		/* add potential last carry */
-	return (0xffff & ~sum);
+	return (~sum & 0xffff);
```

Comment:
This commit is correct, but it's also unnecessary. Unless there's a really good reason for it, I'd drop it.